### ISSUE TITLE
Nothing was uploaded to bintray

### ISSUE THREAD

**Issue by yuyakaido** [novoda/bintray-release#75]
## Environment
- com.novoda:bintray-release:0.3.5
## Detail

I try to publish my android library, run this command. (It is same written in README.)

```
./gradlew clean build bintrayUpload -PbintrayUser=BINTRAY_USERNAME -PbintrayKey=BINTRAY_KEY -PdryRun=false
```

The build has been successful, but nothing was uploaded to bintray. The console log at this time is below.

```
Publication maven not found in project :android-toolkit.
:android-toolkit:clean
:sample:clean
:android-toolkit:compileLint
:android-toolkit:copyDebugLint UP-TO-DATE
:android-toolkit:mergeDebugProguardFiles UP-TO-DATE
:android-toolkit:preBuild UP-TO-DATE
:android-toolkit:preDebugBuild UP-TO-DATE
:android-toolkit:checkDebugManifest
:android-toolkit:prepareDebugDependencies
:android-toolkit:compileDebugAidl
:android-toolkit:compileDebugRenderscript
:android-toolkit:generateDebugBuildConfig
:android-toolkit:generateDebugAssets UP-TO-DATE
:android-toolkit:mergeDebugAssets
:android-toolkit:generateDebugResValues UP-TO-DATE
:android-toolkit:generateDebugResources
:android-toolkit:packageDebugResources
:android-toolkit:processDebugManifest
:android-toolkit:processDebugResources
:android-toolkit:generateDebugSources
:android-toolkit:processDebugJavaRes UP-TO-DATE
:android-toolkit:compileDebugJava
:android-toolkit:packageDebugJar
:android-toolkit:compileDebugNdk UP-TO-DATE
:android-toolkit:packageDebugJniLibs UP-TO-DATE
:android-toolkit:packageDebugLocalJar UP-TO-DATE
:android-toolkit:packageDebugRenderscript UP-TO-DATE
:android-toolkit:bundleDebug
:android-toolkit:compileDebugSources
:android-toolkit:assembleDebug
:android-toolkit:copyReleaseLint UP-TO-DATE
:android-toolkit:mergeReleaseProguardFiles UP-TO-DATE
:android-toolkit:preReleaseBuild UP-TO-DATE
:android-toolkit:checkReleaseManifest
:android-toolkit:prepareReleaseDependencies
:android-toolkit:compileReleaseAidl
:android-toolkit:compileReleaseRenderscript
:android-toolkit:generateReleaseBuildConfig
:android-toolkit:generateReleaseAssets UP-TO-DATE
:android-toolkit:mergeReleaseAssets
:android-toolkit:generateReleaseResValues UP-TO-DATE
:android-toolkit:generateReleaseResources
:android-toolkit:packageReleaseResources
:android-toolkit:processReleaseManifest
:android-toolkit:processReleaseResources
:android-toolkit:generateReleaseSources
:android-toolkit:processReleaseJavaRes UP-TO-DATE
:android-toolkit:compileReleaseJava
:android-toolkit:packageReleaseJar
:android-toolkit:compileReleaseNdk UP-TO-DATE
:android-toolkit:packageReleaseJniLibs UP-TO-DATE
:android-toolkit:packageReleaseLocalJar UP-TO-DATE
:android-toolkit:packageReleaseRenderscript UP-TO-DATE
:android-toolkit:bundleRelease
:android-toolkit:compileReleaseSources
:android-toolkit:assembleRelease
:android-toolkit:assemble
:android-toolkit:lint
Ran lint on variant release: 1 issues found
Ran lint on variant debug: 1 issues found
Wrote HTML report to file:/Users/yuyakaido/Development/Android/AndroidToolkit/android-toolkit/build/outputs/lint-results.html
Wrote XML report to /Users/yuyakaido/Development/Android/AndroidToolkit/android-toolkit/build/outputs/lint-results.xml
:android-toolkit:preCompileDebugUnitTestJava
:android-toolkit:preDebugUnitTestBuild UP-TO-DATE
:android-toolkit:prepareDebugUnitTestDependencies
:android-toolkit:processDebugUnitTestJavaRes UP-TO-DATE
:android-toolkit:compileDebugUnitTestJava UP-TO-DATE
:android-toolkit:compileDebugUnitTestSources UP-TO-DATE
:android-toolkit:mockableAndroidJar
:android-toolkit:assembleDebugUnitTest
:android-toolkit:testDebug
:android-toolkit:preCompileReleaseUnitTestJava
:android-toolkit:preReleaseUnitTestBuild UP-TO-DATE
:android-toolkit:prepareReleaseUnitTestDependencies
:android-toolkit:processReleaseUnitTestJavaRes UP-TO-DATE
:android-toolkit:compileReleaseUnitTestJava UP-TO-DATE
:android-toolkit:compileReleaseUnitTestSources UP-TO-DATE
:android-toolkit:assembleReleaseUnitTest
:android-toolkit:testRelease
:android-toolkit:test
:android-toolkit:check
:android-toolkit:build
:sample:preBuild UP-TO-DATE
:sample:preDebugBuild UP-TO-DATE
:sample:checkDebugManifest
:sample:preReleaseBuild UP-TO-DATE
:sample:prepareAndroidToolkitAndroidToolkitUnspecifiedLibrary
:sample:prepareComAndroidSupportAppcompatV72210Library
:sample:prepareComAndroidSupportSupportV42210Library
:sample:prepareDebugDependencies
:sample:compileDebugAidl
:sample:compileDebugRenderscript
:sample:generateDebugBuildConfig
:sample:generateDebugAssets UP-TO-DATE
:sample:mergeDebugAssets
:sample:generateDebugResValues UP-TO-DATE
:sample:generateDebugResources
:sample:mergeDebugResources
:sample:processDebugManifest
:sample:processDebugResources
:sample:generateDebugSources
:sample:processDebugJavaRes UP-TO-DATE
:sample:compileDebugJava
:sample:compileDebugNdk UP-TO-DATE
:sample:compileDebugSources
:sample:preDexDebug
:sample:dexDebug
:sample:validateDebugSigning
:sample:packageDebug
:sample:zipalignDebug
:sample:assembleDebug
:sample:checkReleaseManifest
:sample:prepareReleaseDependencies
:sample:compileReleaseAidl
:sample:compileReleaseRenderscript
:sample:generateReleaseBuildConfig
:sample:generateReleaseAssets UP-TO-DATE
:sample:mergeReleaseAssets
:sample:generateReleaseResValues UP-TO-DATE
:sample:generateReleaseResources
:sample:mergeReleaseResources
:sample:processReleaseManifest
:sample:processReleaseResources
:sample:generateReleaseSources
:sample:processReleaseJavaRes UP-TO-DATE
:sample:compileReleaseJava
:sample:compileReleaseNdk UP-TO-DATE
:sample:compileReleaseSources
:sample:lintVitalRelease SKIPPED
:sample:preDexRelease
:sample:dexRelease
:sample:packageRelease
:sample:assembleRelease
:sample:assemble
:sample:compileLint
:sample:lint
Ran lint on variant debug: 5 issues found
Ran lint on variant release: 5 issues found
Wrote HTML report to file:/Users/yuyakaido/Development/Android/AndroidToolkit/sample/build/outputs/lint-results.html
Wrote XML report to /Users/yuyakaido/Development/Android/AndroidToolkit/sample/build/outputs/lint-results.xml
:sample:preCompileDebugUnitTestJava
:sample:preDebugUnitTestBuild UP-TO-DATE
:sample:prepareDebugUnitTestDependencies
:sample:processDebugUnitTestJavaRes UP-TO-DATE
:sample:compileDebugUnitTestJava UP-TO-DATE
:sample:compileDebugUnitTestSources UP-TO-DATE
:sample:mockableAndroidJar
:sample:assembleDebugUnitTest
:sample:testDebug
:sample:preCompileReleaseUnitTestJava
:sample:preReleaseUnitTestBuild UP-TO-DATE
:sample:prepareReleaseUnitTestDependencies
:sample:processReleaseUnitTestJavaRes UP-TO-DATE
:sample:compileReleaseUnitTestJava UP-TO-DATE
:sample:compileReleaseUnitTestSources UP-TO-DATE
:sample:assembleReleaseUnitTest
:sample:testRelease
:sample:test
:sample:check
:sample:build
:android-toolkit:bintrayUpload
:android-toolkit:bintrayUpload: Could not find publication: maven.

BUILD SUCCESSFUL

Total time: 30.442 secs
```
## Workaround

I use `com.novoda:bintray-release:0.3.4`, everything works great. The console log at this time is below.

```
:android-toolkit:clean
:sample:clean
:android-toolkit:compileLint
:android-toolkit:copyDebugLint UP-TO-DATE
:android-toolkit:mergeDebugProguardFiles UP-TO-DATE
:android-toolkit:preBuild UP-TO-DATE
:android-toolkit:preDebugBuild UP-TO-DATE
:android-toolkit:checkDebugManifest
:android-toolkit:prepareDebugDependencies
:android-toolkit:compileDebugAidl
:android-toolkit:compileDebugRenderscript
:android-toolkit:generateDebugBuildConfig
:android-toolkit:generateDebugAssets UP-TO-DATE
:android-toolkit:mergeDebugAssets
:android-toolkit:generateDebugResValues UP-TO-DATE
:android-toolkit:generateDebugResources
:android-toolkit:packageDebugResources
:android-toolkit:processDebugManifest
:android-toolkit:processDebugResources
:android-toolkit:generateDebugSources
:android-toolkit:processDebugJavaRes UP-TO-DATE
:android-toolkit:compileDebugJava
:android-toolkit:packageDebugJar
:android-toolkit:compileDebugNdk UP-TO-DATE
:android-toolkit:packageDebugJniLibs UP-TO-DATE
:android-toolkit:packageDebugLocalJar UP-TO-DATE
:android-toolkit:packageDebugRenderscript UP-TO-DATE
:android-toolkit:bundleDebug
:android-toolkit:compileDebugSources
:android-toolkit:assembleDebug
:android-toolkit:copyReleaseLint UP-TO-DATE
:android-toolkit:mergeReleaseProguardFiles UP-TO-DATE
:android-toolkit:preReleaseBuild UP-TO-DATE
:android-toolkit:checkReleaseManifest
:android-toolkit:prepareReleaseDependencies
:android-toolkit:compileReleaseAidl
:android-toolkit:compileReleaseRenderscript
:android-toolkit:generateReleaseBuildConfig
:android-toolkit:generateReleaseAssets UP-TO-DATE
:android-toolkit:mergeReleaseAssets
:android-toolkit:generateReleaseResValues UP-TO-DATE
:android-toolkit:generateReleaseResources
:android-toolkit:packageReleaseResources
:android-toolkit:processReleaseManifest
:android-toolkit:processReleaseResources
:android-toolkit:generateReleaseSources
:android-toolkit:processReleaseJavaRes UP-TO-DATE
:android-toolkit:compileReleaseJava
:android-toolkit:packageReleaseJar
:android-toolkit:compileReleaseNdk UP-TO-DATE
:android-toolkit:packageReleaseJniLibs UP-TO-DATE
:android-toolkit:packageReleaseLocalJar UP-TO-DATE
:android-toolkit:packageReleaseRenderscript UP-TO-DATE
:android-toolkit:bundleRelease
:android-toolkit:compileReleaseSources
:android-toolkit:assembleRelease
:android-toolkit:assemble
:android-toolkit:lint
Ran lint on variant debug: 1 issues found
Ran lint on variant release: 1 issues found
Wrote HTML report to file:/Users/yuyakaido/Development/Android/AndroidToolkit/android-toolkit/build/outputs/lint-results.html
Wrote XML report to /Users/yuyakaido/Development/Android/AndroidToolkit/android-toolkit/build/outputs/lint-results.xml
:android-toolkit:preCompileDebugUnitTestJava
:android-toolkit:preDebugUnitTestBuild UP-TO-DATE
:android-toolkit:prepareDebugUnitTestDependencies
:android-toolkit:processDebugUnitTestJavaRes UP-TO-DATE
:android-toolkit:compileDebugUnitTestJava UP-TO-DATE
:android-toolkit:compileDebugUnitTestSources UP-TO-DATE
:android-toolkit:mockableAndroidJar
:android-toolkit:assembleDebugUnitTest
:android-toolkit:testDebug
:android-toolkit:preCompileReleaseUnitTestJava
:android-toolkit:preReleaseUnitTestBuild UP-TO-DATE
:android-toolkit:prepareReleaseUnitTestDependencies
:android-toolkit:processReleaseUnitTestJavaRes UP-TO-DATE
:android-toolkit:compileReleaseUnitTestJava UP-TO-DATE
:android-toolkit:compileReleaseUnitTestSources UP-TO-DATE
:android-toolkit:assembleReleaseUnitTest
:android-toolkit:testRelease
:android-toolkit:test
:android-toolkit:check
:android-toolkit:build
:sample:preBuild UP-TO-DATE
:sample:preDebugBuild UP-TO-DATE
:sample:checkDebugManifest
:sample:preReleaseBuild UP-TO-DATE
:sample:prepareAndroidToolkitAndroidToolkitUnspecifiedLibrary
:sample:prepareComAndroidSupportAppcompatV72210Library
:sample:prepareComAndroidSupportSupportV42210Library
:sample:prepareDebugDependencies
:sample:compileDebugAidl
:sample:compileDebugRenderscript
:sample:generateDebugBuildConfig
:sample:generateDebugAssets UP-TO-DATE
:sample:mergeDebugAssets
:sample:generateDebugResValues UP-TO-DATE
:sample:generateDebugResources
:sample:mergeDebugResources
:sample:processDebugManifest
:sample:processDebugResources
:sample:generateDebugSources
:sample:processDebugJavaRes UP-TO-DATE
:sample:compileDebugJava
:sample:compileDebugNdk UP-TO-DATE
:sample:compileDebugSources
:sample:preDexDebug
:sample:dexDebug
:sample:validateDebugSigning
:sample:packageDebug
:sample:zipalignDebug
:sample:assembleDebug
:sample:checkReleaseManifest
:sample:prepareReleaseDependencies
:sample:compileReleaseAidl
:sample:compileReleaseRenderscript
:sample:generateReleaseBuildConfig
:sample:generateReleaseAssets UP-TO-DATE
:sample:mergeReleaseAssets
:sample:generateReleaseResValues UP-TO-DATE
:sample:generateReleaseResources
:sample:mergeReleaseResources
:sample:processReleaseManifest
:sample:processReleaseResources
:sample:generateReleaseSources
:sample:processReleaseJavaRes UP-TO-DATE
:sample:compileReleaseJava
:sample:compileReleaseNdk UP-TO-DATE
:sample:compileReleaseSources
:sample:lintVitalRelease SKIPPED
:sample:preDexRelease
:sample:dexRelease
:sample:packageRelease
:sample:assembleRelease
:sample:assemble
:sample:compileLint
:sample:lint
Ran lint on variant debug: 5 issues found
Ran lint on variant release: 5 issues found
Wrote HTML report to file:/Users/yuyakaido/Development/Android/AndroidToolkit/sample/build/outputs/lint-results.html
Wrote XML report to /Users/yuyakaido/Development/Android/AndroidToolkit/sample/build/outputs/lint-results.xml
:sample:preCompileDebugUnitTestJava
:sample:preDebugUnitTestBuild UP-TO-DATE
:sample:prepareDebugUnitTestDependencies
:sample:processDebugUnitTestJavaRes UP-TO-DATE
:sample:compileDebugUnitTestJava UP-TO-DATE
:sample:compileDebugUnitTestSources UP-TO-DATE
:sample:mockableAndroidJar
:sample:assembleDebugUnitTest
:sample:testDebug
:sample:preCompileReleaseUnitTestJava
:sample:preReleaseUnitTestBuild UP-TO-DATE
:sample:prepareReleaseUnitTestDependencies
:sample:processReleaseUnitTestJavaRes UP-TO-DATE
:sample:compileReleaseUnitTestJava UP-TO-DATE
:sample:compileReleaseUnitTestSources UP-TO-DATE
:sample:assembleReleaseUnitTest
:sample:testRelease
:sample:test
:sample:check
:sample:build
:android-toolkit:generatePomFileForMavenPublication
:android-toolkit:mavenAndroidJavadocs
:android-toolkit:mavenAndroidJavadocsJar
:android-toolkit:mavenAndroidSourcesJar
:android-toolkit:publishMavenPublicationToMavenLocal
:android-toolkit:bintrayUpload

BUILD SUCCESSFUL

Total time: 32.664 secs
```

Does anyone know the correct solution?

**Comment by ouchadam** [comment]
thanks for raising this! (great amount of info btw). Looks like https://github.com/novoda/bintray-release/pull/74 may have introduced an issue or the usage might have changed, I'll look into it.

**Comment by mediavrog** [comment]
+1 same here

**Comment by lapism** [comment]
0.3.4 not working for me. Gradle 2.10 and 2.0.0-beta4

**Comment by ouchadam** [comment]
this should be fixed with the `0.4.0` release, let me know if not! 😄

**Comment by andreilisun** [comment]
Same thing happened with 0.5.0. Updated to 0.4.0 and works perfectly.
